project.clj

```diff
@@ -5,7 +5,11 @@
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.10.0"]
                  [org.clojure/java.jmx "0.3.1"]
-                 [jepsen "0.2.1" :exclusions [org.slf4j/slf4j-api]]
+                 [com.github.mwiede/jsch "0.2.18"]
+                 [jepsen "0.2.4" :exclusions [org.slf4j/slf4j-api
+                                              com.jcraft/jsch
+                                              com.jcraft/jsch.agentproxy.core
+                                              ]]
```

Comment:
Can you put the closing brackets on the last line for consistency?

Reply:
changed, even that I'm not totally understand the the importance of it